auth_pysaml/models/__init__.py

```diff
@@ -0,0 +1,8 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from . import auth_saml
+from . import auth_saml_attribute_mapping
+from . import res_users
+from . import saml_token
+from . import saml_request
```

Comment:
Same here, model name:

```suggestion
from . import auth_saml_request
```